contracts/contracts/StakingEscrow.sol

```diff
@@ -287,6 +319,34 @@ contract StakingEscrow is Upgradeable, IERC900History {
         emit Withdrawn(msg.sender, _value);
     }
 
+    /**
+    * @notice Wraps all tokens and top up stake in T staking contract
+    */
+    function wrapAndTopUp() external onlyStaker {
+        StakerInfo storage info = stakerInfo[msg.sender];
+        require(info.stakingProvider != address(0), "There is no stake in T staking contract");
+        require(
+            tStaking.stakedNu(info.stakingProvider) == 0,
```

Comment:
Is it possible that `stakedNu()` will be removed at some point?

Reply:
yep, it will be in second upgrade for threshold staking contract for removing legacy